Puppetfile

```diff
@@ -39,6 +39,12 @@ mod 'puppetlabs/reboot', '2.2.0'
 mod 'puppet/python', '3.0.1'
 mod 'derdanne/nfs', '2.1.2'
 
+# for mate
```

Comment:
Since we're not including the `mate` class, let's drop the module and the dependencies from this file.

Reply:
python module isn't being used by anything else?